fmskill/observation.py

```diff
@@ -252,6 +261,24 @@ def __repr__(self):
         out = f"TrackObservation: {self.name}, n={self.n_points}"
         return out
 
+    def _make_index_unique(self, df):
```

Comment:
Is there a test to verify that this works as expected?

I.e. taking an existing file, rounding the time to get duplicate time entries, and then this operation, and the results should  be comparable?

Reply:
see https://github.com/DHI/fmskill/pull/45/commits/df05f42b896276c2716be5817227d52755a2754d

Reply:
will do

Reply:
Done